### PR TITLE
mobile: Add GetSign to mobile BigInt

### PR DESCRIPTION
**Pull request by aleph7**
Currently, the only way of getting the sign of a BigInt from mobile is
coverting to string. This adds a GetSign function to make conversion
faster.

### CODE REVIEW

**Comment by GitCop** [comment]
Thank you for your contribution! Your commits seem to not adhere to the repository coding standards

 * Commit: 503844e08d5de2d290f1c792687e2c97ea1b2006
  * Commits must be prefixed with the package(s) they modify

Please check the [contribution guidelines](https://github.com/ethereum/go-ethereum#contribution) for more details.

---
This message was auto-generated by https://gitcop.com

**Comment by aleph7** [comment]
@karalabe ping

File: mobile/big.go [hunk 1]
```diff
@@ -62,6 +62,11 @@ func (bi *BigInt) SetInt64(x int64) {
 	bi.bigint.SetInt64(x)
 }
 
+// GetSign returns -1 if x is negative, 0 if x is 0, or 1 if x is positive.
+func (bi *BigInt) GetSign() int {
```
Comment (holiman) [inline mobile/big.go]: Why not just call it `Sign()`, and just reuse the parent docs: 
```
// Sign returns:
//
//	-1 if x <  0
//	 0 if x == 0
//	+1 if x >  0
//
func (bi *BigInt) Sign() int{
	return bi.bigint.Sign()
}
```

**Comment by GitCop** [comment]
Thank you for your contribution! Your commits seem to not adhere to the repository coding standards

 * Commit: 503844e08d5de2d290f1c792687e2c97ea1b2006
  * Commits must be prefixed with the package(s) they modify

 * Commit: efb75b26adade096ea746e2829561bdd359f46b7
  * Commits must be prefixed with the package(s) they modify

Please check the [contribution guidelines](https://github.com/ethereum/go-ethereum#contribution) for more details.

---
This message was auto-generated by https://gitcop.com

**Comment by aleph7** [comment]
Done

**Pull request merged**